js/PKPaymentButton/index.js

```diff
@@ -41,7 +47,8 @@ export class PKPaymentButton extends React.Component<Props> {
   static defaultProps = {
     buttonStyle: "black",
     buttonType: "plain",
```

Comment:
Do we need default values if these are required parameters? Or should they no longer be required if these are sensible defaults?

Reply:
While we don't technically need these default values here, I think it helps to clarify what props we are actually sending the native component. Like you noticed in your [comment](https://github.com/appfolio/react-native-payments/pull/17#discussion_r1612041859) above, we are setting `type = PKPaymentButtonTypePlain` and `style = PKPaymentButtonStyleBlack` on the native component by default to prevent errors in the native layer, and setting the default values to the same thing in this component makes it more apparent that this is what's happening under the hood.